scania-digital-design-system/sdds
# feat(form): added form test story

**Describe pull-request**  
Adds a form test story.

**Solving issue**  
Fixes: [AB#3154](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/3154)

**How to test**  
1. Go to Examples -> Form
2. Try submitting the form and check the console for logs.

## Component to add

- [x] Textfield
- [x] Textarea
- [x] Toggle
- [x] Radio button - added but needs some fixes
- [x] Datepicker
